xbox360controller/controller.py

```diff
@@ -422,7 +422,13 @@ def num_buttons(self):
     def name(self):
         buf = array("B", [0] * 64)
         ioctl(self._dev_file, JSIOCGNAME(len(buf)), buf)
-        return buf.tostring().decode()
+        try:
+            # Python >=3.2
+            buf_bytes = buf.tobytes()
+        except AttributeError:
+            # Python <3.2
+            buf_bytes = buf.tostring()
```

Comment:
Thoughtful but not necessary - Python 3.2 has been EOL for years, that's not worth supporting anymore :)

This can just be an unconditional `return buf.tobytes().decode()`.

Reply:
Changed accordingly